typings/vscode-mssql.d.ts

```diff
@@ -1155,12 +1163,149 @@ declare module 'vscode-mssql' {
 		createStreamingJobTsql: string;
 	}
 
+	export const enum SchemaCompareEndpointType {
+		Database = 0,
+		Dacpac = 1,
+		Project = 2,
+		// must be kept in-sync with SchemaCompareEndpointType in SQL Tools Service
+		// located at \src\Microsoft.SqlTools.ServiceLayer\SchemaCompare\Contracts\SchemaCompareRequest.cs
+	}
+
+	export interface ConnectionInfo {
```

Comment:
We already a class in connectionManager.ts called ConnectionInfo.  Can you rename this to "SchemaCompareConnectionInfo" to make sure we avoid any confusion?

Reply:
I'll change the name to avoid confusion.